src/authentication/components/user-subscription.ts

```diff
@@ -0,0 +1,79 @@
+import { EventEmitter } from 'events'
```

Comment:
There's no React components in this file. Does it belong to another directory, something like `src/authentication/ui`?